octodns/provider/dyn.py

```diff
@@ -709,3 +726,6 @@ def _apply(self, plan):
         self._apply_regular(desired, changes, dyn_zone)
 
         dyn_zone.publish()
+
+        if self.cleanup_session:
+            self._close_dyn_sess()
```

Comment:
Wouldn't the caches need to be invalidate here too?

Reply:
Was relying on `publish()` to flush the cache in this case:
https://github.com/markrechler/octodns/blob/dyn_sessions/octodns/provider/dyn.py#L81